packages/cashier/src/stores/general-store.ts

```diff
@@ -103,57 +91,6 @@ export default class GeneralStore extends BaseStore {
         return !!currency && isCryptocurrency(currency);
     }
 
-    get is_p2p_enabled(): boolean {
-        const { content_flag } = this.root_store.traders_hub;
-        const is_eu = [ContentFlag.LOW_RISK_CR_EU, ContentFlag.EU_REAL].includes(content_flag);
-
-        return this.is_p2p_visible && !is_eu;
-    }
-
-    /**
-     * Gets the notifications from local storage, within p2p_settings, where it checks which notification has
-     * been seen. The number of unseen notifications is displayed in vertical tab, notifications count, for P2P.
-     *
-     * @returns {number} Notifications that have not been seen by the user.
-     */
-    get p2p_unseen_notifications(): number {
```

Comment:
Seems to be unused, Please let me know if we shouldn't remove it 🙇🏻🙏🏻

Reply:
Yup this is fine to remove, i have seen that you have replaced all the instances of it with the new hook 👍